src/components/LandingPage/LandingPage.js

```diff
@@ -14,8 +14,70 @@ const LandingPage = (props) => {
         + "unclassified training and education courses, seminars, instructional"
         + " resources and more."
     const placeholderText = '';
-    // TODO: replace this json obj with a real call to elasticsearch
-    const dummy_json = dummyJSON;
+
+    // const api_url = process.env.REACT_APP_ES_MLT_API;
+    const api_url = "http://localhost:8080/es-api/spotlight-courses";
```

Comment:
If you have this in your .env.local you can now refer to it using process.env.<name>;